app/lib/frontend/dom/dom.dart

```diff
@@ -32,6 +32,9 @@ abstract class DomContext {
 
   /// Creates a DOM Text node.
   Node text(String value);
+
+  /// Creates a DOM node with safe HTML content.
+  Node safeHtml(String value);
```

Comment:
This method doesn't sanitize html in any way... I think it's better to call this `rawHtml` rather than `safeHtml`.

The `rawHtmlString` string passed to `dom.rawHtml(rawHtmlString)` should be sanitized.

Reply:
Just saying, this is a way to inject raw HTML, right?

Calling it `safeHtml` makes me think that it's "safe" to inject any HTML here...

Perhaps a better name is `rawUnsafeHtml` or ´unsafeRawHtml`, or something like that.. the method is inherently unsafe, right?

Reply:
Good point, renamed.

Reply:
One alternative would be to create `Node.markdownContent(String markdownSource)`, which would inside ensure it is safe... although it would need to accept our custom hooks...

Reply:
I've renamed it to `rawUnsafeHtml`, but `unsafeRawHtml` may sound better if I read out the code. Let me know if you prefer the later.